src/Components/TicketSelector/DestinationDesk.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import DeskMenu from "./DeskMenu";
+
+import { GrClose } from "react-icons/gr";
+import { theme } from "../../styles/theme";
+
+class DestinationDesk extends Component {
+  constructor() {
+    super();
+    this.state = {
+      location: {},
+    };
+  }
+
+  componentDidMount() {
+    fetch("/data/data.json")
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          location: res.location,
+        });
+      });
+  }
+
+  closeModal = () => {
+    this.props.closeDestination();
+  };
+
+  render() {
+    const { location } = this.state;
+    const {
+      top,
+      left,
+      closeDestination,
+      selectDepPlace,
+      selectArrPlace,
+    } = this.props;
+
+    return (
+      <Desk top={top} left={left}>
+        <DeskContainer>
+          <Headers>
+            <span>도시 선택</span>
+            <span onClick={this.closeModal}>
+              <GrClose />
+            </span>
+          </Headers>
+          <Main>
+            <MainTop>
+              <input type="text" placeholder="도시명을 입력하세요" />
+              <button>검색</button>
+            </MainTop>
+            <MainBottom>
+              <div className="bottomHeader">주요도시 바로 선택</div>
+              {location.length && (
```

Comment:
```suggestion
              {location.length > 0 && (
```
이렇게 작성하시거나
```suggestion
              {!!location.length && (
```
이렇게 작성하시면 됩니다.

0일 경우 && 우측의 내용이 실행되지는 않겠지만 대신 화면에 0 자체가 표시되어버리기 때문에 boolean 값으로 변환시켜야 하기 때문입니다.

Reply:
원인이 이거였군요... 수정 하겠습니다!!